### PR TITLE
Updating some translations for Brazilian Portuguese

### PR DESCRIPTION
**Pull request by Draacoun**
Fixing translations errors, message formats and more.

### CODE REVIEW

File: src/main/resources/assets/iris/lang/pt_br.json [hunk 1]
```diff
@@ -1,66 +1,68 @@
...
-  "iris.shaders.debug.enabled": "A depuração foi habilitada, o log de depuração será mais detalhado!",
-  "iris.shaders.debug.disabled": "A depuração foi desabilitada.",
-  "iris.shaders.debug.failure": "Não é possível habilitar a depuração, seu computador não suporta recursos de depuração.",
-  "iris.shaders.debug.restart": "Você não tem um contexto de depuração, reinicie para habilitar a depuração.",
-  "iris.keybind.reload": "Recarregar Shaders",
-  "iris.keybind.shaderPackSelection": "Tela de Seleção de Shaderpack",
-  "iris.keybind.toggleShaders": "Alternar Shaders",
+  "iris.shaders.reloaded": "Sombreadores recarregados!",
+  "iris.shaders.disabled": "Sombreadores desativados!",
+  "iris.shaders.toggled": "Sombreadores alternado para %s!",
```
Comment (rafaeeelv) [inline src/main/resources/assets/iris/lang/pt_br.json:4]:
```suggestion
  "iris.shaders.toggled": "Sombreadores alternados para %s!",
```

File: src/main/resources/assets/iris/lang/pt_br.json [hunk 2]
```diff
@@ -1,66 +1,68 @@
...
+  "iris.shaders.debug.enabled": "A depuração foi ativada, seus registros serão mais detalhados!",
+  "iris.shaders.debug.disabled": "A depuração foi desativada.",
+  "iris.shaders.debug.failure": "Não foi possível ativar a depuração pois seu computador não suporta os recursos de depuração.",
```
Comment (rafaeeelv) [inline src/main/resources/assets/iris/lang/pt_br.json:7]:
```suggestion
  "iris.shaders.debug.failure": "Não foi possível ativar a depuração pois seu computador não suporta os recursos necessários.",
```

**Review by rafaeeelv — CHANGES_REQUESTED** [review]

**Comment by IMS212** [comment]
Would you mind rebasing this on trunk?

**Comment by Draacoun** [comment]
> Would you mind rebasing this on trunk?

Well, I'm don't use GitHub too often so I don't think I made the right change in this pull request. 😅 

The correct order to rebase this for trunk branch is in fact the edit button in the title? Sorry if this is a dumb question, I really don't know.
image

**Pull request closed without merging**